.github/workflows/release.yml

```
@@ -23,7 +23,7 @@ on:
 
 jobs:
   release:
-    if: github.event.pull_request.merged == true
+    if: ${{github.event_name == 'workflow_dispatch' || github.event.pull_request.merged}}
```

```suggestion
    if: ${{github.event_name != 'pull_request' || github.event.pull_request.merged}}
```
Just because `event.pull_request.merged` only exists for pull_request events. Doesn't change the functionality in this case, but might do if we want to trigger a new release via a different trigger in the future.